### PR TITLE
SRT-14: update global contants

### PR DESCRIPTION
**Pull request by FrozenIce0617**
- Add global constants
- Remove unused yarn.lock
- Add `.prettierrc` file for the VSCode

### CODE REVIEW

File: src/css/vars.scss [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
 $brass: #AC6E16;
+$black: #0A0806;
+$white: #FFFFFF;
+$grey-light: #f2f2f2;
 $grey-10: #F4F4F4;
 $grey-20: #E6E6E6;
 $grey-30: #c4c4c4;
 $grey-40: #AAAAAA;
-$grey-50: #818181;
+$grey-50: #818181;
```
Comment (maymax777) [inline src/css/vars.scss:19]: Please remove the `space` at the end of the line that can affect the lint issue for the deploy.

**Comment by maymax777** [comment]
LGTM 👍

**Review by micahjlucas — CHANGES_REQUESTED** [review]
Code looks good - just need to fix merge conflicts.

**Review by micahjlucas — APPROVED** [review]
Actually, it was a small merge conflict, so I just fixed. Looks great!

**Pull request merged**